### DISCUSSION TITLE
An option to hide the "[upgrade] is active!" messages?

### DISCUSSION THREAD

**Discussion by someoneelse06** [Malcolm-Q/LC-LateGameUpgrades#376] [Ideas]
When you have several of them enabled, they tend to crowd the screen. It'd be nice being able to hide them.

**Comment by WhiteSpike** [comment]
I can add a config option to not let them appear. I was thinking of later making it like a popup which fades out where it shows the loaded upgrades + benefits from them though I would need time to learn some UI.
